modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -5764,8 +5764,10 @@ export const console: ConsoleNS = {
                             validationErrorMessages: {
                                 duplicateName: "The {{type}} attribute already exists.",
                                 invalidName: "The name you entered contains illegal characters. " +
-                                    "Only alphabets, numbers, `#`, and `_` are allowed."
-                            }
+                                    "Only alphabets, numbers, `#`, and `_` are allowed.",
```

Comment:
What's the use of `#` here?

Reply:
This the error message for local attributes which allows to have `#`